src/RMKERR.CPP

```diff
@@ -563,9 +566,13 @@ RC errI(		// report error common inner fcn for errCrit, errV
 	const char* text;
 	if (erOp & SYSMSG)
 	{
+		#if !defined( ELIMINATE_WINDOWS)
```

Comment:
I'd like to understand a bit more of the context here before we just cut out this code.

Reply:
@chipbarnaby, after looking at this a bit closer I'm not sure we'll ever hit an instance with a SYSMSG. Maybe we can remove this code entirely?